vendor/k8s.io/utils/mount/mount_helper_common.go

```diff
@@ -73,9 +73,13 @@ func doCleanupMountPoint(mountPath string, mounter Interface, extensiveMountPoin
 		return err
 	}
 
-	notMnt, mntErr := mounter.IsLikelyNotMountPoint(mountPath)
-	if mntErr != nil {
-		return mntErr
+	if extensiveMountPointCheck {
```

Comment:
cc @saad-ali on other storage changes that came in with the utils bump

Reply:
ref https://github.com/kubernetes/kubernetes/issues/89181